docs/build/beginners_guide/src/docs/_OneDayTutorial.adoc

```diff
@@ -43,7 +43,7 @@ include::_TutorialIntro.adoc[]
 //--- tutorial steps --------------------------------------------------------//
 
 [[sec-contacts_initial_project]]
-=== Setting up the Initial Project
```

Comment:
We write titles in "Title Case", so using an uppercase letter here is correct (https://www.grammar-monster.com/lessons/capital_letters_title_case.htm)